README.rst

```diff
@@ -10,9 +10,9 @@ SeleniumLibrary_ is a web testing library for `Robot Framework`_ that
 utilizes the Selenium_ tool internally. The project is hosted on GitHub_
 and downloads can be found from PyPI_.
 
-SeleniumLibrary works with Selenium 3 and 4. It supports Python 2.7 as well as
```

Comment:
Could you this file as it is for while. It will be updated when 5.0 is release is made.

Reply:
Reverted!